storage/shard.go

```diff
@@ -893,6 +881,16 @@ func (s *dbShard) FlushState(blockStart time.Time) fileOpState {
 	return state
 }
 
+func (s *dbShard) markFlushStateSuccessOrError(blockStart time.Time, err error) error {
```

Comment:
It seems all use of "fluid interface" tends to be returning this/self https://en.wikipedia.org/wiki/Fluent_interface